eZ/Publish/Core/REST/Client/LocationService.php

```diff
@@ -80,16 +81,20 @@ public function setSession($id)
      * Instantiates a new location create class.
      *
      * @param mixed $parentLocationId the parent under which the new location should be created
+     * @param eZ\Publish\API\Repository\Values\ContentType\ContentType|null $contentType
      *
      * @return \eZ\Publish\API\Repository\Values\Content\LocationCreateStruct
      */
-    public function newLocationCreateStruct($parentLocationId)
+    public function newLocationCreateStruct($parentLocationId, ContentType $contentType = null)
     {
-        return new LocationCreateStruct(
-            array(
-                'parentLocationId' => $parentLocationId,
-            )
-        );
+        $properties = [
+            'parentLocationId' => $parentLocationId,
+        ];
+        if ($contentType) {
+            $properties['sortField'] = $contentType->defaultSortField;
+            $properties['sortOrder'] = $contentType->defaultSortOrder;
+        }
+        return new LocationCreateStruct($properties);
```

Comment:
hmm, seems CS checked for some reason did not spot this _(cs conf or travis conf not up to date on 6.7?)_
But afaik our CS profile dictates a empty line before return statements, right @alongosz ?